rpcs3/util/serialization_ext.cpp

```diff
@@ -1253,7 +1251,6 @@ void compressed_zstd_serialization_file_handler::stream_data_prepare_thread_op()
 			break;
 		}
 
-		usz buffer_offset = 0;
```

Comment:
Remove whitespace too

Reply:
lol what?

Reply:
You didnt remove the newline, there are 2 consequives newlines.

Reply:
What are you talking about?

Reply:
lol, just look at the code here at your text editor.